### DISCUSSION TITLE
Sind die Seiten und TAB Body Stil JSON nicht bindingfähig?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2021] [Q&A]
![image](https://user-images.githubusercontent.com/84818934/199588450-1688deea-67b0-4356-988c-9405c612194c.png)

`{"'{0_userdata.0.jarvis.test.forumTestDP5}'='bad'":{"display":"none !important"}}`

![image](https://user-images.githubusercontent.com/84818934/199588536-63e4c050-fb9d-45a9-93f2-c93aaf796a14.png)

ioBroker-Objekt:
![image](https://user-images.githubusercontent.com/84818934/199588589-caf453de-735c-460a-ba57-e5386a4ab9a4.png)

Objekte Seite und TAB weiterhin zu sehen?
![image](https://user-images.githubusercontent.com/84818934/199589556-681c3398-7b8c-4b7c-9245-8134701c9e95.png)

![image](https://user-images.githubusercontent.com/84818934/199589716-b288586a-7093-41db-8ba8-f36690ab1211.png)

Zeitweise wird so angezeigt:
![image](https://user-images.githubusercontent.com/84818934/199761457-d0509926-dd79-455c-b328-affd3d6a3e26.png)

![image](https://user-images.githubusercontent.com/84818934/199761503-2eb7f0b0-c5cc-4ec0-b393-b7a998e7c5f4.png)

**Comment by Zefau** [comment]
Ja, sollte gehen

**Reply by Zefau** [reply]
Nachverfolgung via https://github.com/Zefau/ioBroker.jarvis/issues/2040
